INFO:tensorflow:Using default config.
WARNING:tensorflow:Using temporary folder as model directory: /tmp/tmpvv8s3g81
INFO:tensorflow:Using config: {'_model_dir': '/tmp/tmpvv8s3g81', '_tf_random_seed': None, '_save_summary_steps': 100, '_save_checkpoints_steps': None, '_save_checkpoints_secs': 600, '_session_config': allow_soft_placement: true
graph_options {
  rewrite_options {
    meta_optimizer_iterations: ONE
  }
}
, '_keep_checkpoint_max': 5, '_keep_checkpoint_every_n_hours': 10000, '_log_step_count_steps': 100, '_train_distribute': None, '_device_fn': None, '_protocol': None, '_eval_distribute': None, '_experimental_distribute': None, '_service': None, '_cluster_spec': <tensorflow.python.training.server_lib.ClusterSpec object at 0x7f76757b6278>, '_task_type': 'worker', '_task_id': 0, '_global_id_in_cluster': 0, '_master': '', '_evaluation_master': '', '_is_chief': True, '_num_ps_replicas': 0, '_num_worker_replicas': 1}
WARNING:tensorflow:From /home/rafaelbraga25/LEI/Privacy/env/lib/python3.7/site-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
WARNING:tensorflow:From /home/rafaelbraga25/LEI/Privacy/env/lib/python3.7/site-packages/tensorflow_estimator/python/estimator/inputs/queues/feeding_queue_runner.py:62: QueueRunner.__init__ (from tensorflow.python.training.queue_runner_impl) is deprecated and will be removed in a future version.
Instructions for updating:
To construct input pipelines, use the `tf.data` module.
WARNING:tensorflow:From /home/rafaelbraga25/LEI/Privacy/env/lib/python3.7/site-packages/tensorflow_estimator/python/estimator/inputs/queues/feeding_functions.py:500: add_queue_runner (from tensorflow.python.training.queue_runner_impl) is deprecated and will be removed in a future version.
Instructions for updating:
To construct input pipelines, use the `tf.data` module.
INFO:tensorflow:Calling model_fn.
WARNING:tensorflow:From /home/rafaelbraga25/LEI/Privacy/env/lib/python3.7/site-packages/tensorflow/python/keras/layers/core.py:143: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
WARNING:tensorflow:From /home/rafaelbraga25/LEI/Privacy/env/lib/python3.7/site-packages/tensorflow/python/ops/array_grad.py:425: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
/home/rafaelbraga25/LEI/Privacy/env/lib/python3.7/site-packages/tensorflow/python/ops/gradients_impl.py:110: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Graph was finalized.
2019-06-13 22:14:00.414546: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2019-06-13 22:14:00.419588: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2601000000 Hz
2019-06-13 22:14:00.419752: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x55627b593300 executing computations on platform Host. Devices:
2019-06-13 22:14:00.419773: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
OMP: Info #212: KMP_AFFINITY: decoding x2APIC ids.
OMP: Info #210: KMP_AFFINITY: Affinity capable, using global cpuid leaf 11 info
OMP: Info #154: KMP_AFFINITY: Initial OS proc set respected: 0
OMP: Info #156: KMP_AFFINITY: 1 available OS procs
OMP: Info #157: KMP_AFFINITY: Uniform topology
OMP: Info #159: KMP_AFFINITY: 1 packages x 1 cores/pkg x 1 threads/core (1 total cores)
OMP: Info #214: KMP_AFFINITY: OS proc to physical thread map:
OMP: Info #171: KMP_AFFINITY: OS proc 0 maps to package 0 
OMP: Info #250: KMP_AFFINITY: pid 47737 tid 47737 thread 0 bound to OS proc set 0
2019-06-13 22:14:00.420019: I tensorflow/core/common_runtime/process_util.cc:71] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:From /home/rafaelbraga25/LEI/Privacy/env/lib/python3.7/site-packages/tensorflow/python/training/monitored_session.py:809: start_queue_runners (from tensorflow.python.training.queue_runner_impl) is deprecated and will be removed in a future version.
Instructions for updating:
To construct input pipelines, use the `tf.data` module.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 0 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
OMP: Info #250: KMP_AFFINITY: pid 47737 tid 47751 thread 1 bound to OS proc set 0
OMP: Info #250: KMP_AFFINITY: pid 47737 tid 47752 thread 2 bound to OS proc set 0
INFO:tensorflow:loss = 2.3053133, step = 1
INFO:tensorflow:Saving checkpoints for 58 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 2.0359526.
INFO:tensorflow:Calling model_fn.
WARNING:tensorflow:From /home/rafaelbraga25/LEI/Privacy/env/lib/python3.7/site-packages/tensorflow/python/ops/metrics_impl.py:455: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-13T21:15:39Z
INFO:tensorflow:Graph was finalized.
WARNING:tensorflow:From /home/rafaelbraga25/LEI/Privacy/env/lib/python3.7/site-packages/tensorflow/python/training/saver.py:1266: checkpoint_exists (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to check for files with this prefix.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-58
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-13-21:15:39
INFO:tensorflow:Saving dict for global step 58: accuracy = 0.6015, global_step = 58, loss = 2.0261915
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 58: /tmp/tmpvv8s3g81/model.ckpt-58
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-58
WARNING:tensorflow:From /home/rafaelbraga25/LEI/Privacy/env/lib/python3.7/site-packages/tensorflow/python/training/saver.py:1070: get_checkpoint_mtimes (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file utilities to get mtimes.
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 58 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 2.0297222, step = 59
INFO:tensorflow:Saving checkpoints for 116 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.8202729.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-13T21:17:22Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-116
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-13-21:17:22
INFO:tensorflow:Saving dict for global step 116: accuracy = 0.7766, global_step = 116, loss = 1.797263
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 116: /tmp/tmpvv8s3g81/model.ckpt-116
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-116
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 116 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.8550639, step = 117
INFO:tensorflow:Saving checkpoints for 174 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.7706006.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-13T21:19:08Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-174
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-13-21:19:09
INFO:tensorflow:Saving dict for global step 174: accuracy = 0.7999, global_step = 174, loss = 1.7300954
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 174: /tmp/tmpvv8s3g81/model.ckpt-174
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-174
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 174 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.7359108, step = 175
INFO:tensorflow:Saving checkpoints for 232 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.7051483.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-13T21:20:59Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-232
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-13-21:20:59
INFO:tensorflow:Saving dict for global step 232: accuracy = 0.8126, global_step = 232, loss = 1.6967179
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 232: /tmp/tmpvv8s3g81/model.ckpt-232
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-232
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 232 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.695205, step = 233
INFO:tensorflow:Saving checkpoints for 290 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:From /home/rafaelbraga25/LEI/Privacy/env/lib/python3.7/site-packages/tensorflow/python/training/saver.py:966: remove_checkpoint (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to delete files with this prefix.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.6456498.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-13T21:22:52Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-290
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-13-21:22:53
INFO:tensorflow:Saving dict for global step 290: accuracy = 0.8844, global_step = 290, loss = 1.6420964
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 290: /tmp/tmpvv8s3g81/model.ckpt-290
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-290
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 290 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.6678079, step = 291
INFO:tensorflow:Saving checkpoints for 348 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.6384699.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-13T21:24:50Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-348
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-13-21:24:51
INFO:tensorflow:Saving dict for global step 348: accuracy = 0.891, global_step = 348, loss = 1.620892
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 348: /tmp/tmpvv8s3g81/model.ckpt-348
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-348
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 348 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.6353743, step = 349
INFO:tensorflow:Saving checkpoints for 406 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.6195812.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-13T21:26:52Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-406
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-13-21:26:52
INFO:tensorflow:Saving dict for global step 406: accuracy = 0.8971, global_step = 406, loss = 1.6088135
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 406: /tmp/tmpvv8s3g81/model.ckpt-406
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-406
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 406 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.6279056, step = 407
INFO:tensorflow:Saving checkpoints for 464 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.566652.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-13T21:28:58Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-464
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-13-21:28:58
INFO:tensorflow:Saving dict for global step 464: accuracy = 0.9005, global_step = 464, loss = 1.5992548
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 464: /tmp/tmpvv8s3g81/model.ckpt-464
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-464
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 464 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.6064422, step = 465
INFO:tensorflow:Saving checkpoints for 522 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.6057721.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-13T21:31:07Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-522
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-13-21:31:08
INFO:tensorflow:Saving dict for global step 522: accuracy = 0.9033, global_step = 522, loss = 1.5926796
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 522: /tmp/tmpvv8s3g81/model.ckpt-522
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-522
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 522 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.5974298, step = 523
INFO:tensorflow:Saving checkpoints for 580 into /tmp/tmpvv8s3g81/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.5849745.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-13T21:33:21Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpvv8s3g81/model.ckpt-580
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-13-21:33:22
INFO:tensorflow:Saving dict for global step 580: accuracy = 0.9066, global_step = 580, loss = 1.5877323
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 580: /tmp/tmpvv8s3g81/model.ckpt-580

WARNING: The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
If you depend on functionality not listed there, please file an issue.

For delta=1e-5, the current epsilon is: 1.56
Test accuracy after 1 epochs is: 0.601
For delta=1e-5, the current epsilon is: 1.73
Test accuracy after 2 epochs is: 0.777
For delta=1e-5, the current epsilon is: 1.89
Test accuracy after 3 epochs is: 0.800
For delta=1e-5, the current epsilon is: 2.03
Test accuracy after 4 epochs is: 0.813
For delta=1e-5, the current epsilon is: 2.17
Test accuracy after 5 epochs is: 0.884
For delta=1e-5, the current epsilon is: 2.30
Test accuracy after 6 epochs is: 0.891
For delta=1e-5, the current epsilon is: 2.42
Test accuracy after 7 epochs is: 0.897
For delta=1e-5, the current epsilon is: 2.54
Test accuracy after 8 epochs is: 0.900
For delta=1e-5, the current epsilon is: 2.66
Test accuracy after 9 epochs is: 0.903
For delta=1e-5, the current epsilon is: 2.77
Test accuracy after 10 epochs is: 0.907
Elapsed time:  1162.1489281654358
